src/node/desktop/package.json

```diff
@@ -70,8 +70,9 @@
     "crc": "^3.8.0",
     "electron-squirrel-startup": "^1.0.0",
     "electron-store": "^8.0.1",
+    "i18next": "^21.6.10",
```

Comment:
Add i18next to https://github.com/rstudio/rstudio/blob/main/src/node/desktop/NOTICE.electron

The license info is here: https://github.com/i18next/i18next/blob/master/LICENSE

I've been adding them in alphabetical order but it doesn't really matter.